src/pool.rs

```diff
@@ -560,6 +575,20 @@ impl<Conn: Connection + Send + 'static> Pool<Conn> {
     /// Acquires a handle to a connection within the connection pool.
     #[instrument(level = "debug", skip(self), err, name = "Pool::claim")]
     pub async fn claim(&self) -> Result<claim::Handle<Conn>, Error> {
+        #[cfg(feature = "probes")]
+        let id = usdt::UniqueId::new();
+        #[cfg(feature = "probes")]
+        probes::claim__start!(|| &id);
```

Comment:
do we want information about the Pool in here?

Reply:
I think I'm going to punt on this until we find we need a way to distinguish multiple pools in a single process.